perfkitbenchmarker/linux_packages/openssl.py

```diff
@@ -24,3 +24,32 @@ def YumInstall(vm):
 def AptInstall(vm):
   """Installs OpenSSL on the VM."""
   vm.InstallPackages('openssl libssl-dev')
+
+
+def AptInstallQAT(vm):
+  """Installs QAT engine for accelerated OpenSSL."""
+
+  vm.InstallPackages('autoconf build-essential libtool cmake cpuid libssl-dev pkg-config nasm')
+  
+  # Install Intel® Integrated Performance Primitives Cryptography lib  
+  vm.RemoteCommand('git clone https://github.com/intel/ipp-crypto.git')
+  vm.RemoteCommand('cd ipp-crypto')
+  vm.RemoteCommand('git checkout ippcp_2021.7.1')
```

Comment:
Every remote command starts a new shell session. Therefore, you are back in home directory here and cannot directly call cmake.
To fix, merge line 36 and 37 by doing
cd cd ipp-crypto && git checkout ippcp_2021.7.1